README.md

```diff
@@ -44,30 +44,30 @@ You should install the software listed below based on your testing purpose, incl
     |Debugging and testing|Testing tools core features|Required|Required|
     |Developent Activities|Visual Studio C++ core features||Required<sup>1</sup>|
     |SDKs, libraries, and frameworks|Windows 10 SDK (10.0.16299.0) for Desktop C++ [x86 and x64]||Required<sup>1</sup>|
-1. [Spec Explorer 2010 v3.5.3146.0](https://visualstudiogallery.msdn.microsoft.com/271d0904-f178-4ce9-956b-d9bfa4902745/)
+1. [Spec Explorer 2010 v3.5.3146.0](https://visualstudiogallery.msdn.microsoft.com/271d0904-f178-4ce9-956b-d9bfa4902745/)<br />
    It is only required if you want to build or run the test suites that contain Model-Based Test cases. If you want to regenerate Model-Based Test cases, you must install Visual Studio 2012.
-1. [Protocol Test Framework build 1.0 (build 1.0.6500.0)](https://github.com/Microsoft/ProtocolTestFramework/releases/tag/1.0.6500.0)
+1. [Protocol Test Framework build 1.0 (build 1.0.6500.0)](https://github.com/Microsoft/ProtocolTestFramework/releases/tag/1.0.6500.0)<br />
    You can use a released MSI file or build it from source code.
-1. Enable .NET Framework 3.5.1
-   _Turn Windows features on or off_
-   ☑ _.NET Framework 3.5 (includes .NET 2.0 and 3.0)
+1. Enable .NET Framework 3.5.1<br/>
```

Comment:
[Enable .NET Framework 3.5.1]  instead

Reply:
Do you mean:

    1. Enable .NET Framework 3.5.1<br/>
       1. _Turn Windows features on or off_
       1. check _.NET Framework 3.5 (includes .NET 2.0 and 3.0)_
       This is necessary for WiX Toolset.

?

Reply:
ping @sweeetday

Reply:
Hi, Jtmoon79
Yes, the short set is preferred.
Thanks.